src/app/(proper_react)/redesign/(public)/Faq.tsx

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactNode, useState } from "react";
+import { useL10n } from "../../../hooks/l10n";
+import styles from "./LandingView.module.scss";
+import { CloseBigIcon } from "../../../components/server/Icons";
+import Link from "next/link";
+
+export type FaqItemProps = {
+  question: string;
+  answer: string | ReactNode;
+  isExpanded: boolean;
+  onExpandAnswer: () => void;
+};
+
+const FaqItem = (props: FaqItemProps) => {
+  const { question, answer, isExpanded, onExpandAnswer } = props;
+
+  return (
+    <>
+      <dt className={styles.faqQuestion} onClick={onExpandAnswer}>
+        {question}
+        <CloseBigIcon alt="" className={`${isExpanded && styles.expanded}`} />
+      </dt>
+      <dd className={`${styles.faqAnswer} ${isExpanded && styles.expanded}`}>
+        {answer}
+      </dd>
+    </>
+  );
+};
+
+export const FaqSection = ({
+  isEligibleForPremium,
+}: {
+  isEligibleForPremium: boolean;
+}) => {
+  const l10n = useL10n();
+  const [expandedQuestion, setExpandedQuestion] = useState<string | null>(null);
+
+  const handleExpandAnswer = (question: string) => {
+    setExpandedQuestion((prevQuestion) =>
+      prevQuestion === question ? null : question,
+    );
+  };
+
+  return (
+    <div className={styles.faqWrapper}>
+      <b className={styles.faqTitle}>
+        {l10n.getString("landing-all-faq-title")}
+      </b>
+      <Link
+        className={styles.faqCta}
+        href={process.env.NEXT_PUBLIC_FAQ_MONITOR as string}
+        target="_blank"
+      >
+        {l10n.getString("landing-all-faq-see-all")}
+      </Link>
```

Comment:
I thought [Next.js `<Link>`](https://nextjs.org/docs/pages/api-reference/components/link) was mainly for internal routing and not external links.
https://upmostly.com/nextjs/how-to-link-to-an-external-page-in-nextjs